src/viam/sdk/common/exception.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
+#include <grpc/status.h>
+#include <grpcpp/impl/codegen/status.h>
+#include <iostream>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+
+namespace viam {
+namespace sdk {
+
+enum class ViamErrorCode : uint8_t {
+    // Used to indicate no error
+    ok = 0,
+    generic = 1,
```

Comment:
What does `generic` mean here? I think `unknown` is more typical as a catch-all from what I've seen.

Reply:
Good idea, `unknown` is clearer.